Teclast/P20HD/AndroidManifest.xml

```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.teclast.P20HD"
+        android:versionCode="1"
+        android:versionName="1.0">
+        <overlay android:targetPackage="android"
+                android:requiredSystemPropertyName="ro.vendor.build.fingerprint"
+		android:requiredSystemPropertyValue="+Teclast/P20HD_EEA/P20HD_EEA*"
```

Comment:
I'm just now sure if I wrote this correctly.

```console
➜ adb shell getprop ro.vendor.build.fingerprint
Teclast/P20HD_EEA/P20HD_EEA:10/QP1A.190711.020/26621:user/release-keys
```

Reply:
@phhusson have you seen the comment above? I know you merged already, but I'm just double-checking to ensure I didn't make any mistake.

Reply:
Oh, I just realized the typo now. But yes, that's what I meant.

Thank you then!